templates/leadership/provost-signature.php

```diff
@@ -0,0 +1,26 @@
+
+<tr>
+  <td class="ccollapse100">
+    <table class="tcollapse100" style="width: 100%;">
+      <tbody>
+        <tr>
+          <td class="ccollapse100" valign="top" style="width: 88px; text-align: left; padding-bottom: 15px;" width="88px;">
+            <img alt="UCF Provost" src="<?php echo UCF_Email_Editor_Config::get_option_or_default( 'provost_signature_image' ); ?>" style="display: inline-block; padding-right: 20px;">
+          </td>
+          <td class="ccollapse100">
+            <table class="tcollapse100" style="width: 100%;">
+              <tbody>
+                <tr>
+                  <td class="montserratbold" style="width: 100%; font-size: 16px; font-family: Helvetica, Arial, sans-serif; font-weight: bold; padding-bottom: 2px;">Elizabeth A. Dooley</td>
+                </tr>
+                <tr>
+                  <td class="montserrat" style="width: 100%; font-size: 16px; font-family: Helvetica, Arial, sans-serif; font-weight: normal; padding-bottom: 2px;">Provost and Vice President for Academic Affairs</td>
```

Comment:
Updating `<td class="montserrat"` to `<td class="montserratlight"` would fix the missing font face on the job title here